Invalid escape sequences

```
./django_libs/decorators.py:56:30: W605 invalid escape sequence '\w'
./django_libs/decorators.py:56:33: W605 invalid escape sequence '\.'
./django_libs/models.py:10:47: W605 invalid escape sequence '\.'
./django_libs/models.py:10:55: W605 invalid escape sequence '\.'
./django_libs/test_email_backend.py:51:25: W605 invalid escape sequence '\.'
```

--- a/django_libs/decorators.py
+++ b/django_libs/decorators.py
@@ -53,7 +53,7 @@ def lockfile(lockfile_name, lock_wait_timeout=-1):
 
 def get_username(identifier):
     """Checks if a string is a email adress or not."""
-    pattern = re.compile('.+@\w+\..+')
+    pattern = re.compile(r'.+@\w+\..+')
     if pattern.match(identifier):
         try:
             user = User.objects.get(email=identifier)

--- a/django_libs/models.py
+++ b/django_libs/models.py
@@ -7,7 +7,7 @@ try:
 except ImportError:
     pass
 else:
-    add_introspection_rules([], ["^django_libs\.models\.ColorField"])
+    add_introspection_rules([], [r"^django_libs\.models\.ColorField"])
 
 from .widgets import ColorPickerWidget
 

--- a/django_libs/test_email_backend.py
+++ b/django_libs/test_email_backend.py
@@ -40,7 +40,7 @@ class EmailBackend(SmtpEmailBackend):
 
 
 class WhitelistEmailBackend(SmtpEmailBackend):
-    """
+    r"""
     Email backend that sends only these emails, that match the whitelist
     setting.
 

